### ISSUE TITLE
Annoying notifications

### ISSUE THREAD

**Issue by dadoprom** [nitroshare/nitroshare-desktop#104]
Please add option to "disable notification" into settings... On my linux box (gnome) these notifications keep on poping up - so so annoying, and I did not find any way how to turn them off. And it is all about "machine xy has joined"...

**Comment by nathan-osman** [comment]
Yes, this should definitely be an option. I'll make sure this gets into the next release.

**Comment by no-username42** [comment]
when will be the time for the 'next release'?
I just started using nitroshare and got at least 20 notifications in one day. It´s really distracting.

**Comment by nathan-osman** [comment]
The next release is proving to be a massive undertaking, and the work completed to date is enormous.

However, the daily builds are considerably more stable than they were before, so if you are feeling brave, you can give them a shot. The PPA is here:

https://launchpad.net/~george-edison55/+archive/ubuntu/nitroshare-dev

The new version uses its own logging system and does not yet have any means of displaying notifications. Once that is added, it will offer options for fine-tuning what is displayed as well as an option to disable it altogether.

**Comment by x9sim9** [comment]
Could you point me in the right direction as to where notifications are in the current version, and I can do a quick fork to remove notification

**Comment by x9sim9** [comment]
For now anyone who is annoyed by this with windows 10 can disable notifications from Nitroshare in
Settings -> System -> Notifications & Actions -> Network File Transfer Application

**Comment by TilTisback** [comment]
> 
> 
> For now anyone who is annoyed by this with windows 10 can disable notifications from Nitroshare in
> Settings -> System -> Notifications & Actions -> Network File Transfer Application

thanks for this workaround!
